setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = construction_survey_results
+description = Project for construction survey
+version = 0.0.1
+author = Office for National Statistics
+platforms = win32
+classifiers =
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+packages =
+    cons_results
+install_requires =
+    pyyaml
+    pandas
+    numpy
+    monthly-business-survey-results @ git+https://github.com/ONSdigital/monthly-business-survey-results.git@v0.1.1
```

Comment:
just monthly-business-survey-results  will install the latest version available of mbs version deployed in artifactory ?

Reply:
Would work internally, but workflows cant connect and get packages from there so unit tests would fail on PR

Reply:
Oh you are right forgot about this, there must be a command to install the latest release from GitHub or we want to pin it ?

Reply:
Spend some time this afternoon trying to point it towards the latest release but couldnt get anywhere. I'll open an issue about this and just merge for now. It will need to be manually updated as we go on but works for now